cotobang/src/test/java/com/example/cotobang/fixture/UserFixtureFactory.java

```diff
@@ -0,0 +1,29 @@
+package com.example.cotobang.fixture;
+
+import com.example.cotobang.dto.UserRegistrationDto;
+
+public class UserFixtureFactory {
+
+    public UserRegistrationDto createUserRegistrationDto() {
```

Comment:
테스트니까 한국어 메소드 이름을 사용해보는 것도 좋은 경험이 될 거에요.

```suggestion
    public UserRegistrationDto create_사용자_등록_DTO() {
```